src/experiments/preferences/api.js

```diff
@@ -56,13 +37,26 @@ var preferences = class preferences extends ExtensionAPI {
           async getIntPref(name, defaultValue) {
             return Services.prefs.getIntPref(name, defaultValue);
           },
+          async setIntPref(name, defaultValue) {
+            return Services.prefs.setIntPref(name, defaultValue);
+          },
```

Comment:
I don't want to block this project, I'm just not happy with this. In general, I really dislike exposing an api that could be used to set any pref in the system.  The API should have been written to provide an api that internally set the specific prefs we're dealing with.